package-lock.json

```diff
@@ -6197,7 +6197,8 @@
     "fsevents": {
       "version": "2.2.1",
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-2.2.1.tgz",
-      "integrity": "sha512-bTLYHSeC0UH/EFXS9KqWnXuOl/wHK5Z/d+ghd5AsFMYN7wIGkUCOJyzy88+wJKkZPGON8u4Z9f6U4FdgURE9qA=="
+      "integrity": "sha512-bTLYHSeC0UH/EFXS9KqWnXuOl/wHK5Z/d+ghd5AsFMYN7wIGkUCOJyzy88+wJKkZPGON8u4Z9f6U4FdgURE9qA==",
+      "optional": true
```

Comment:
Please remove the change to the package-lock.json file from this PR (in general you should be able to avoid having unrelated changes to the package-lock.json file by using `npm ci` if the PR doesn't require changes to the dependencies listed in the package.json file).